python_modules/dagster/dagster/core/instance/__init__.py

```diff
@@ -1243,6 +1244,15 @@ def get_run_records(
     def supports_bucket_queries(self):
         return self._run_storage.supports_bucket_queries
 
+    @traced
+    def get_run_partition_data(
+        self, partition_set_name: str, job_name: str, repository_label: str
```

Comment:
i had imagined this would be a selector, but i guess as per the previous diff we ended up combining repo and location into one, so this seems reasonable.. the number of different ways we have in python to identify a job is getting quite large though. Might be worth commenting what this label is in the storage class.

The alternative would be to have the graphql layer parse the label into repo + location, and have this method be more selector-y

Reply:
yeah, I did have a repository selector here at first, and then I started worrying that I was mixing concepts across layers too much.  But maybe it doesn't matter anyway, considering the fact that we're storing origin_id / selector_id already?  I dunno, it felt weird to import those types in.